resources/assets/sass/app.scss

```diff
@@ -311,6 +312,11 @@ span.user-array > input {
   padding: 2em 4em;
 }
 
+.inactive {
+    color: #757575;
```

Comment:
is there an ARC colour in the variables file that we can use?

Reply:
I looked at `$arc_gray` but that's seems lighter than what you're trying to do here

Reply:
I was going for "lighter", so $arc_gray could be used. Opinion on if it's ok as is, or should reuse the existing?

Reply:
if you're pushing changes to this commit might as well be consistent and use the css variables. I guess it's also kind of agreed with ARC to use those colours (brand specific maybe?)

Reply:
OK, i'll switch it out.

Reply:
OK! so `$arc_gray` is the background colour 😂 leaving as #757575